scripts/post-checkout

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+SCRIPT_PATH="$(dirname "$0")/$(dirname "$(readlink "$0")")"
+
+# Import utils
+source ${SCRIPT_PATH}/utils/message
+NODE_VERSION=$(node -v)
+
+if [ "$(node -p -e "require('./package.json').devDependencies.node")" != "undefined" ]; then
+    if [ "$(echo $NODE_VERSION | | cut -d. -f1)" -lt 6 ]; then
```

Comment:
As far as I understand after looking at it for 10 minutes this checks for the major version. Could you move this into a variable? `$MAJOR_NODE_VERSION -lt 6` would be nicer to read IMHO